### ISSUE TITLE
Checker - In High contrast White theme, more option button of Scan is not visible.

### ISSUE THREAD

**Issue by marcjohlic** [IBMa/equal-access#1106]
- [X] `a11y Checker`
- IBM Accessibility checker Version 3.1.37
- Keyboard FF Windows 

Page element: More Option button
All the elements should be clearly visible in High Contrast themes.

Impact statement: 
When the element is not visible on high contrast themes, low vision users who uses High Contrast will face difficulty in accessing the application.

WCAG2.1 mapping: 
4.1.4 Accessibility-supported Technologies Only (IBM requirement)

Triage: SME to review and see recommend a fix. 

From Ind Bug - Moved from internal board https://github.ibm.com/IBMa/E2E/issues/4997

**Comment by drjoho** [comment]
Ok. Carbon Overflow menu Dark Theme makes button black with white icon and white border for affordance. The white theme makes button white with black icon and a blue border for affordance. Neither of these work. Currently we have a black button, white chevron, and a blue border as specified by design.

This issue should go to design before dev.

**Comment by drjoho** [comment]
Fixed in #945
